src/core/public/chrome/chrome_service.tsx

```diff
@@ -431,6 +447,13 @@ export interface ChromeStart {
    */
   setBreadcrumbs(newBreadcrumbs: ChromeBreadcrumb[]): void;
 
+  /**
+   * Mount an element next to the last breadcrumb
+   */
+  setBreadcrumbsAppendExtension(
```

Comment:
New extension point in chrome to add a session status indicator next to the last breadcrumb. 

![Screenshot 2020-10-28 at 16 29 24](https://user-images.githubusercontent.com/7784120/97459974-9d234a80-193c-11eb-86c5-d6156786fd83.png)

Reply:
The `Append` part of the name inspired by EUI: https://elastic.github.io/eui/#/forms/form-controls#prepend-and-append. I assume in future we could add `Prepend` 
The `Extension` part is named similar to `HelpExtension`